eu-north-1/dev/ec2.tf

```diff
@@ -0,0 +1,108 @@
+#import vpc module
+#import vpc module
+module "vpc_module" {
```

Comment:
This will create another vpc of the same name, conflict.
It's better to use module to reference the VPC created inside the `vpc.tf` one way.
`module.vpc_module.id`